apps/ledger-live-mobile/src/screens/Nft/hooks/useChains.test.ts

```diff
@@ -0,0 +1,104 @@
+import useFeature from "@ledgerhq/live-common/featureFlags/useFeature";
+import useEnv from "@ledgerhq/live-common/hooks/useEnv";
+
+import { useSelector } from "react-redux";
+import { useChains } from "./useChains";
+import { renderHook } from "@testing-library/react-native";
+
+jest.mock("@ledgerhq/live-common/featureFlags/useFeature");
+jest.mock("@ledgerhq/live-common/hooks/useEnv");
+jest.mock("react-redux", () => ({
+  useSelector: jest.fn(),
+}));
```

Comment:
For `useFeature` and `useSelector` why not using `renderHook` from "@tests/test-renderer" and override the initial state.

Reply:
Indeed. I'll update that !Did saw the import was wrong aha